src/org/ods/orchestration/usecase/LeVADocumentUseCase.groovy

```diff
@@ -983,11 +985,21 @@ class LeVADocumentUseCase extends DocGenUseCase {
                 documentHistory: docHistory?.getDocGenFormat() ?: [],
             ]
         ]
+
+        if (calculateIfProjectIsGxp(this.project.getProjectProperties())) {
+            data_.data.isGxpProject = true;
```

Comment:
@valituguran perhaps is more readable:
`data_.data.isGxpProject = calculateIfProjectIsGxp(this.project.getProjectProperties())`
In this way the data.isGxpProject always exists with a value passed to DocGen service.

Reply:
@hrcornejo agreed, but I did not sent it intentionally when false because in the templates we are checking for existance of that field. 
We can modify it like this and also update the templates side.

Reply:
@hrcornejo the changes are pushed now.

Reply:
Thanks!!! @valituguran